lib/kupo/manager.rb

```diff
@@ -0,0 +1,63 @@
+module Kupo
+  class Manager
```

Comment:
Dropping the `config` and making this `Pharos:SSH::Manager` would also make sense.

Then this change would just essentially be about removing the pseudo-global `Pharos::SSH` class methods to prevent any thread-unsafe usage - which is indeed the point, using explicit "dependency injection" for the `@ssh` in the phases...